cloudinit/config/schemas/schema-cloud-config-v1.json

```diff
@@ -230,6 +230,101 @@
       },
       "additionalProperties": true
     },
+    "cc_ansible": {
+      "type": "object",
+      "properties": {
+        "ansible": {
+          "type": "object",
+          "additionalProperties": false,
+          "properties": {
+            "install-method": {
+              "type": "string",
+              "default": "distro",
+              "enum": [
+                "distro",
+                "pip"
+              ],
+              "description": "The type of installation for ansible. It can be one of the following values:\n\n - ``distro``\n - ``pip``\n - ``none``"
```

Comment:
I forgot to remove the null/none option. Since the commands differ between distro and pip installs, I don't think we should allow not installing it (which set of commands would we attempt?). The code currently checks if it is already installed prior to attempting to install, so that if it is installed during a different config module it doesn't repeat the call. Assuming there are no objections to this approach, I'll just update the description.

Reply:
Sorry, I thought it was a valid input. Then remove it.